client/python/samples/README.md

```diff
@@ -222,12 +237,215 @@ optional arguments:
+                                  [--model_version MODEL_VERSION]
+                                  [--timeout TIMEOUT]
+
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --http_address HTTP_ADDRESS Specify url to http service. default:localhost
+  --http_port HTTP_PORT Specify port to http service. default: 9000
+  --model_name MODEL_NAME Model name to query. default: resnet
+  --model_version MODEL_VERSION Model version to query. Lists all versions if omitted
+  --timeout TIMEOUT Request timeout. default: 10.0
+```
+- Usage Example
+
+```
+python get_http_model_metadata.py --http_port 9000 --model_name resnet --model_version 1
+{1: {'inputs': {'map/TensorArrayStack/TensorArrayGatherV3': {'shape': [1, 224, 224, 3], 'dtype': 'DT_FLOAT'}}, 'outputs': {'softmax_tensor': {'shape': [1, 1001], 'dtype': 'DT_FLOAT'}}}}
+```
+
+
+### Predict numeric format <a name="predict-numeric-http">
+
+#### **Make prediction using images in numerical format:**
+
+- Command
+
+```bash
+usage: resnet_http_predict.py [-h] --images_dir IMAGES_DIR
+                              [--http_address HTTP_ADDRESS]
+                              [--http_port HTTP_PORT]
+                              [--model_name MODEL_NAME]
+                              [--model_version MODEL_VERSION]
+                              [--timeout TIMEOUT]
+
+
+optional arguments:
+  -h, --help            show this help message and exit 
+  --images_dir IMAGES_DIR Path to a directory with images in JPG or PNG format
+  --http_address HTTP_ADDRESS Specify url to http service. default:localhost
+  --http_port HTTP_PORT Specify port to http service. default: 9000
+  --model_name MODEL_NAME Model name to query. default: resnet
+  --model_version MODEL_VERSION Model version to query. default: latest available
+  --timeout TIMEOUT Request timeout. default: 10.0
+```
+- Usage example
+
+```
+python resnet_http_predict.py --http_port 9000 --images_dir images --model_name resnet
+Image images/magnetic_compass.jpeg has been classified as magnetic compass with 99.99372959136963% confidence
+Image images/pelican.jpeg has been classified as pelican with 99.17410612106323% confidence
+Image images/gorilla.jpeg has been classified as gorilla, Gorilla gorilla with 98.07604551315308% confidence
+Image images/snail.jpeg has been classified as snail with 99.97051358222961% confidence
+Image images/zebra.jpeg has been classified as zebra with 99.4793951511383% confidence
+Image images/arctic-fox.jpeg has been classified as Arctic fox, white fox, Alopex lagopus with 93.65214705467224% confidence
+Image images/bee.jpeg has been classified as bee with 96.6326653957367% confidence
+Image images/peacock.jpeg has been classified as peacock with 99.97820258140564% confidence
+Image images/airliner.jpeg has been classified as airliner with 49.202319979667664% confidence
+Image images/golden_retriever.jpeg has been classified as golden retriever with 88.68610262870789% confidence
+```
```

Comment:
Can we drop printing the confidence? Some resnets from Open Model Zoo do not come with softmax as a last layer and confidence is not calculated correctly for them in our scripts. To avoid confusion we could just print the result of classification without confidence stats.